util/topgen.py

```diff
@@ -700,7 +704,7 @@ def create_mem(item, addrsep, regwidth):
                          swaccess=swaccess)
 
 
-def _process_top(topcfg, args, cfg_path, out_path, pass_idx):
+def _process_top(topcfg, args, cfg_path, out_path, pass_idx, alt_hjson_path=""):
```

Comment:
As above, all the call sites pass the `alt_hjson_path` argument, so I don't think this should have a default value.

Reply:
yeah i completely didn't notice `args` was already passed in, so this was redundant to start with.